src/ResourceManagement/Network/Microsoft.Azure.Management.Network/Generated/Models/Page1.cs

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+// 
+// Code generated by Microsoft (R) AutoRest Code Generator 0.16.0.0
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+
+namespace Microsoft.Azure.Management.Network.Models
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using Newtonsoft.Json;
+    using Microsoft.Rest.Azure;
+
+    /// <summary>
+    /// Defines a page in Azure responses.
+    /// </summary>
+    /// <typeparam name="T">Type of the page content items</typeparam>
+    [JsonObject]
+    public class Page1<T> : IPage<T>
```

Comment:
@vinatara This comes from the nextLink: null in your x-ms-pageable methods.  If you want to remove it, you will need to set nextLink: to the appropriate value.  If you want to keep this as is, we will need to discuss.